src/integrationTest/java/uk/gov/hmcts/reform/lrdapi/RetrieveCourtVenueDetailsIntegrationTest.java

```diff
@@ -53,7 +53,7 @@ void retrieveCourtVenues_AllepimmsIdGiven_ShouldReturnValidResponseAndStatusCode
                                                          LrdCourtVenueResponse[].class, path
             );
 
-        assertThat(response).isNotEmpty().hasSize(12);
+        assertThat(response).isNotEmpty().hasSize(13);
```

Comment:
add assertion for EXTERNAL_SHORT_NAME value

Reply:
done